IdleConstants.js

```diff
@@ -0,0 +1,29 @@
+const HISTORY_LOG_SIZE = 100;
+
+const SUCCESS_CODE = 200;
+const RESOURCE_LOCKED_CODE = 423;
+const TIER_LEADERBOARD_KEY = 'tieredLeaderboardData';
+const TITLE_ID_TOP_TIER_SUFIX = '_TopTier';
+const TITLE_ID_GLOBAL_SUFIX = '_Global';
+
+const SERVER_CONFIG = {
+        leaderboardServers: {
+                "3D16B": "http://UltrabitWebServicesProd.ya4zvdebvk.us-west-2.elasticbeanstalk.com/", // kongregate/production
+                "19081": "http://uws-dev-env.ddhjvp3mrf.us-west-1.elasticbeanstalk.com/", // staging
+                "18A45": "http://uws-dev-env.ddhjvp3mrf.us-west-1.elasticbeanstalk.com/"  // auto
+        },
+        logglyTag: {
+                "3D16B": "idle-spongebob-kongregate-ops", // kongregate/production
+                "19081": "idle-spongebob-auto-ops", // staging
+                "18A45": "idle-spongebob-auto-ops"  // auto
+        }
+};
+
+const LOGGLY_TOKEN = '00cced27-e9fd-49e2-b332-87d611e71937';
+const DEFAULT_KONG_NAME = "NoKongName";
```

Comment:
we should be save if we move `DEFAULT_KONG_NAME` to `IdleServerPlayerProfile.js`